transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannel.java

```diff
@@ -132,6 +134,10 @@ protected Object filterOutboundMessage(Object msg) {
         return super.filterOutboundMessage(msg);
     }
 
+    public PeerCredentials peerCredentials() throws IOException {
```

Comment:
Done here's what was added.

/*
    * returns the unix credentials (uid, gid, pid) of the peer
    * <a href=http://man7.org/linux/man-pages/man7/socket.7.html>SO_PEERCRED</a>
    *
    * @returns PeerCredentials
    * @throws IOException
    */